lib/gui/app/app.js

```diff
@@ -345,6 +358,20 @@ app.controller('HeaderController', function (OSOpenExternalService) {
   this.shouldShowHelp = () => {
     return !process.env.ETCHER_DISABLE_EXTERNAL_LINKS
   }
+
+  /**
+   * @summary Whether to show WiFi dialog link
+   * @function
+   * @public
+   *
+   * @returns {Boolean}
+   *
+   * @example
+   * HeaderController.shouldShowWifi()
+   */
+  this.shouldShowWifi = () => {
+    return Boolean(process.env.ETCHER_ENABLE_WIFI_DIALOG)
```

Comment:
`Boolean('0')` or `Boolean('false')` is true

Reply:
Good call, believe that's fixed now.